web-registry/src/pages/ProjectLocation/ProjectLocation.tsx

```diff
@@ -65,40 +65,41 @@ const ProjectLocation: React.FC = () => {
 
   async function saveValues(values: ProjectLocationFormValues): Promise<void> {
     const metadata = { ...projectData?.projectById?.metadata, ...values };
-    const feature = values['schema:location'];
-    let addressId = projectData?.projectById?.addressId;
-    // if there's a current address associated with project, update it, otherwise create a new one
-    if (!addressId) {
-      const { data: addressData } = await createAddress({
-        variables: {
-          input: {
-            address: {
-              feature,
-            },
-          },
-        },
-      });
-      addressId = addressData?.createAddress?.address?.id;
-    } else {
-      await updateAddress({
-        variables: {
-          input: {
-            id: addressId,
-            addressPatch: {
-              feature,
-            },
-          },
-        },
-      });
-    }
+    // TODO: Do we need to store addresses anymore? Commented out because the update mutation was failing.
```

Comment:
@S4mmyb Do we still want to store addresses of projects in our off-chain DB? This may have flown under the radar as something we used to do. Maybe keep it as a legacy option?

Reply:
I think we can just pass on doing this for now @mhagel. If users want to do that via the metadata they can, but i don't really see the need or use in storing an address if it isn't going to be posted publically

Reply:
it looks like the address table was a part of building the registry back-end as a purchasing system: https://github.com/regen-network/registry-server/blob/master/sql/V1_19__purchase.sql

Reply:
seeing as we're going with a crypto happy path first on the purchasing front (ie not supporting fiat) I don't think we need to include this

Reply:
OK. will delete the postgres address update in this flow. But we will still have the address table in the DB in case it is needed elsewhere.